supervisor/api/middleware/security.py

```diff
@@ -190,6 +190,7 @@ async def system_validation(
     ) -> Response:
         """Check if core is ready to response."""
         if self.sys_core.state not in (
+            CoreState.SETUP,
```

Comment:
Would it not make sense to also change the message?
`message=f"System is not ready with core state: {self.sys_core.state.value}"`
or
`message=f"System is not ready with state of core: {self.sys_core.state.value}"`

Reply:
I don't really see the difference here.